doc/changes.md

```diff
@@ -1,5 +1,91 @@
+* Figure.subplot: Fix strange positioning issues after exiting subplot ([#2427](https://github.com/GenericMappingTools/pygmt/pull/2427))
+* pygmt.config: Correctly resetting to default values that contain whitespaces ([#2331](https://github.com/GenericMappingTools/pygmt/pull/2331))
+* pygmt.set_display: Do nothing when the display method is set to 'none' ([#2450](https://github.com/GenericMappingTools/pygmt/pull/2450))
+
+### Documentation
+
+* GMTDataArrayAccessor: Add inline examples for setting GMT specific properties (#2370) ([#2370](https://github.com/GenericMappingTools/pygmt/pull/2370))
+* Document limitations of GMT xarray accessors ([#2375](https://github.com/GenericMappingTools/pygmt/pull/2375))
+* Revise the notes about registration and gtype of remote datasets ([#2384](https://github.com/GenericMappingTools/pygmt/pull/2384))
+* Add project keywords to the pyproject.toml file ([#2315](https://github.com/GenericMappingTools/pygmt/pull/2315))
+* Add inline example for colorbar ([#2373](https://github.com/GenericMappingTools/pygmt/pull/2373))
+* Add inline example for grdview ([#2381](https://github.com/GenericMappingTools/pygmt/pull/2381))
+* Add inline example for load_earth_mask ([#2355](https://github.com/GenericMappingTools/pygmt/pull/2355))
+* Add inline example for load_earth_vertical_gravity_gradient ([#2356](https://github.com/GenericMappingTools/pygmt/pull/2356))
+* Add inline examples and improve documentation for pygmt.set_display ([#2458](https://github.com/GenericMappingTools/pygmt/pull/2458))
+* Add gallery example showing how to use patterns via the "fill" parameter (or similiar parameters) ([#2329](https://github.com/GenericMappingTools/pygmt/pull/2329))
+* Add gallery example for scatter plot with histograms on sides ([#2410](https://github.com/GenericMappingTools/pygmt/pull/2410))
+* Add gallery example showing how to use advanced grdgradient via the "azimuth" & "normalize" parameters ([#2354](https://github.com/GenericMappingTools/pygmt/pull/2354))
+* Add gallery example for the Figure.timestamp method ([#2391](https://github.com/GenericMappingTools/pygmt/pull/2391))
+* Expand gallery example "Focal mechanisms" to use "*fill" and "pen" ([#2433](https://github.com/GenericMappingTools/pygmt/pull/2433))
+* Add working example to quickstart section of README ([#2369](https://github.com/GenericMappingTools/pygmt/pull/2369))
+* Recommend Mambaforge and mamba in the installation and contributing guides ([#2385](https://github.com/GenericMappingTools/pygmt/pull/2385))
+
+### Maintenance
+
+* Add the GMTSampleData class to simplify the load_sample_data and list_sample_data functions (#2342) ([#2342](https://github.com/GenericMappingTools/pygmt/pull/2342))
+* Add a new target 'doctest' to run doctests only and simplify Makefile ([#2443](https://github.com/GenericMappingTools/pygmt/pull/2443))
+* Add a package-level variable __gmt_version__ for development use ([#2366](https://github.com/GenericMappingTools/pygmt/pull/2366))
```

Comment:
Need to wrap this in backticks, otherwise it the leading and suffix underscores don't show up because markdown treats it as bold.
```suggestion
* Add a package-level variable `__gmt_version__` for development use ([#2366](https://github.com/GenericMappingTools/pygmt/pull/2366))
```